src/viam/resource/registry.py

```diff
@@ -112,6 +114,28 @@ def register_resource_creator(cls, subtype: "Subtype", model: "Model", creator:
                 raise DuplicateResourceError(key)
             cls._RESOURCES[key] = creator
 
+    @classmethod
+    def register_resource_validator(cls, subtype: "Subtype", model: "Model", validator: "Validator"):
+        """Register a specific validator function
```

Comment:
Some inaccuracies in this doc comment. `creator (ResourceCreator)` arg should be `validator (Validator)` arg. I _think_ this only raises `DuplicateResourceError`. And, there should be no `Returns` section.

Reply:
Oh my, thank you for catching this!